### PR TITLE
Upstreaming  10 - Status

### PR DESCRIPTION
**Pull request by CKolkey**

### CODE REVIEW

**Comment by ten3roberts** [comment]
It seems as github has cached and does not recalculate the diff based on the target branch having merges.

Happened yesterday as well with upstreaming 8, and waiting solved the issue then.

**Comment by CKolkey** [comment]
Yeah, the fix is to rebase onto master, then rebase onto the correct target again.

File: lua/neogit/status.lua [hunk 1]
```diff
@@ -937,6 +980,63 @@ local cmd_func_map = function()
         vim.cmd("split " .. item.name)
       end
     end,
+    ["GoToPreviousHunkHeader"] = function()
```
Comment (ten3roberts) [inline lua/neogit/status.lua:983]: Ok, I *really* like this feature ❤️

**Review by ten3roberts — APPROVED** [review]

File: lua/neogit/status.lua (hunk 1 above)
Comment (CKolkey) [reply lua/neogit/status.lua:983]: :D Yeah! It's _slightly_ wonky when you're at the bottom/top of a file, but thats just an issue of fine-tuning

**Pull request merged**